install/install.sh

```diff
@@ -93,10 +101,28 @@ echo "Checking for docker."
 ## We unset this variable for this command to avoid conflicts with companion version
 docker --version || curl -fsSL https://get.docker.com | env -u VERSION sh || (
     echo "Failed to start docker, something is wrong."
-    echo "Please report this problem."
-    exit 1
+    echo "Are in in CI? trying dind."
 )
 systemctl enable docker
+if [ $RUNNING_IN_CI -eq 1 ]
+then
+
+
+set -x && \
```

Comment:
there are extra spaces here on the beginning and the indentation level is wrong. 

Can't you also use:
```
set -x
    code...
set +x 
``` 
?